node/service/src/overseer.rs

```diff
@@ -186,6 +192,17 @@ where
 	let spawner = SpawnGlue(spawner);
 
 	let builder = Overseer::builder()
+		.network_bridge_tx(NetworkBridgeTxSubsystem::new(
+			network_service.clone(),
+			authority_discovery_service.clone(),
+			Metrics::register(registry)?,
+		))
+		.network_bridge_rx(NetworkBridgeRxSubsystem::new(
+			network_service.clone(),
+			authority_discovery_service.clone(),
+			Box::new(network_service.clone()),
+			Metrics::register(registry)?,
```

Comment:
you're registering the same metrics twice

Reply:
(they need to be split into two different `Metrics` structs)

Reply:
Uhh, good point

Reply:
Fixed